nautobot/docs/models/dcim/deviceredundancygroups.md

```diff
@@ -0,0 +1,359 @@
+# Device Redundancy Groups
```

Comment:
Should probably add a include of this file in `nautobot/docs/core-functionality/devices.md` for discoverability.